concourse/steps/notification.py

```diff
@@ -150,3 +151,22 @@ def resolve_recipients_by_component(components, github_cfg_name):
         components=components,
     )
     return component_recipients
+
+
+def send_slack_notifications(
+    message: str,
+    slack_cfg_name: str,
+    slack_channel: str,
+    subject: str,
+):
+    slack_cfg = ci.util.ctx().cfg_factory().slack(slack_cfg_name)
```

Comment:
pass-in cfg-factory (dependency-injection)

also, can we simply reuse existing slackclient.util-module?